EPRN.Common.Data/Migrations/20240117153148_Add-Category-Field.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace EPRN.Waste.API.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddCategoryField : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "Category",
+                table: "WasteJourney",
+                type: "int",
+                nullable: true);
```

Comment:
I think the category is determined at waste record creation. Therefore this should not be nullable. Please check if this is the case, and change if I am correct

Reply:
Done.